aws/src/test/java/org/apache/iceberg/aws/glue/TestGlueCatalog.java

```diff
@@ -602,17 +601,18 @@ public void testTablePropsDefinedAtCatalogLevel() {
         null,
         catalogProps);
     Map<String, String> properties = glueCatalog.properties();
-    Assert.assertFalse(properties.isEmpty());
-    Assert.assertTrue(properties.containsKey("table-default.key1"));
-    Assert.assertEquals("catalog-default-key1", properties.get("table-default.key1"));
-    Assert.assertTrue(properties.containsKey("table-default.key2"));
-    Assert.assertEquals("catalog-default-key2", properties.get("table-default.key2"));
-    Assert.assertTrue(properties.containsKey("table-default.key3"));
-    Assert.assertEquals("catalog-default-key3", properties.get("table-default.key3"));
-    Assert.assertTrue(properties.containsKey("table-override.key3"));
-    Assert.assertEquals("catalog-override-key3", properties.get("table-override.key3"));
-    Assert.assertTrue(properties.containsKey("table-override.key4"));
-    Assert.assertEquals("catalog-override-key4", properties.get("table-override.key4"));
+    Assertions.assertThat(properties)
+        .isNotEmpty()
+        .containsKey("table-default.key1")
```

Comment:
we can omit all the `containsKey` calls for which we also check `containsEntry`

Reply:
I've omit all the containsKey